packages/react/figma.config.json

```
@@ -0,0 +1,10 @@
+{
+  "codeConnect": {
+    "parser": "react",
+    "importPaths": {
+      "src/*": "@primer/brand"
+    },
+    "include": ["src/**/*.figma.tsx", "src/**/*.tsx"],
+    "exclude": ["test/**", "docs/**", "build/**"]
```

```suggestion
    "exclude": ["src/**/*.stories.tsx", "src/**/*.test.tsx"]
```

The exclusions likely have no impact on this reoi. We should exclude the tests and Storybook stories that are co-located with the code.